cypress/integration/admin_portal/helpers/enterprise_coupons.js

```diff
@@ -0,0 +1,87 @@
+const uuidv4 = require('uuid/v4')
+
+function getDates() {
+  let dates = {}
+  let target_date = new Date()
+  target_date.setDate(target_date.getDate() - 3)
+  dates.start_date = target_date.toISOString()
+  target_date.setDate(target_date.getDate() + 20)
+  dates.end_date = target_date.toISOString()
+  return dates
+}
+
+
+class EnterpriseCoupons {
+
+  loginToEcommerce() {
+    let token
```

Comment:
I am not sure if we really need to declare this variable? what about declaring a variable at class level and than use it every where?